generated/google-apis-storage_v1/lib/google/apis/storage_v1/service.rb

```diff
@@ -2858,6 +2858,8 @@ def get_object_iam_policy(bucket, object, generation: nil, user_project: nil, fi
         #   IO stream or filename containing content to upload
         # @param [String] content_type
         #   Content type of the uploaded content.
+        # @param @param [IO, String] upload_id
```

Comment:
This is an auto generated library. You're not supposed to make manual changes to this.

Reply:
I think instead of making changes here. You can use `options` parameter in the `create_file` method of Storage veneer to pass these variables.